R/ergm.mple.R

```diff
@@ -84,6 +84,10 @@ ergm.mple<-function(nw, fd, m, init=NULL,
                 theta.offset=init,
 		control=control,
                 verbose=verbose)
+  
+  if(control$MPLE.covariance.method != "invHess" && control$MPLE.covariance.method != "Godambe" && control$MPLE.covariance.method != "Bootstrap"){
+    abort("Not a valid covariance method.")
+  }
```

Comment:
This is probably not necessary with the check in `control.ergm()`.

Reply:
That's correct. I have deleted these lines.